contrib/udfs/pom.xml

```diff
@@ -31,10 +31,22 @@
   <name>Drill : Contrib : UDFs</name>
 
   <dependencies>
+    <dependency>
```

Comment:
I think that DNS lookups can be done in the standard JDK without any libraries. But I'm not sure about all of the functions supported by these UDFs.